### PR TITLE
Hide "Try Premium for free" & "For Business"

### PR DESCRIPTION
**Pull request by ZivSimchoni**
Hide "Try Premium for free" is multiple places such: Top bar, Side bar (in home feed), My network and "Messaging"

Hide "For Business" from top bar

| Before | After |
:------:  | :------:
|HomeBefore | HomeAfter  |
|FeedBefore | FeedAfter|

hope that helps (:

### CODE REVIEW

**Comment by mig4ng** [comment]
Hi @ZivSimchoni , first off thanks for creating this PR.
I believe the `Try premium` upsell elements are already hidden by the rules already in place (at least in Firefox).

Check my linkedin right now with the rules available in the repository:
image

Second, I completely let the premium banner pass in the Network tab, thanks for that one. Can you also add the one showing in jobs so we merge it together?

Thank you, I will also comment other things in the code itself.

File: linkedin.txt [hunk 1]
```diff
@@ -11,3 +11,17 @@ www.linkedin.com##.premium-upsell-link--extra-long
 ! Hide LinkedIn messages overlay
 www.linkedin.com##.msg-overlay-container-reflow.msg-overlay-container
 www.linkedin.com##.mb2.artdeco-card.community-panel
+
+! Hide LinkedIn Pay for premium under the profile in Feed page
+www.linkedin.com##.text-align-left.link-without-hover-state.t-bold.t-black.t-12.feed-identity-module__anchored-widget--premium-upsell.feed-identity-module__anchored-widget.link-without-visited-state.app-aware-link
+
+! Hide LinkedIn  "Try Premium for free" top bar
+www.linkedin.com##.premium-upsell-link--truncate.global-nav__spotlight-upsell.global-nav__primary-link--premium.global-nav__primary-link.link-without-visited-state
```
Comment (mig4ng) [inline linkedin.txt]:
```suggestion
```
I think these can be removed. But double check please.

**Review by mig4ng — CHANGES_REQUESTED** [review]

File: linkedin.txt [hunk 2]
```diff
@@ -11,3 +11,17 @@ www.linkedin.com##.premium-upsell-link--extra-long
...
+
+! Hide LinkedIn "Try Premium for free" in My Network ! NOTE: loading this page is slower if enabled
+www.linkedin.com##.relative.flex-column.display-flex.align-items-left.mb4.artdeco-card.p4.mn-sales-navigator-upsell
+www.linkedin.com##.artdeco-card.full-width.mn-abi-form__container
```
Comment (mig4ng) [inline linkedin.txt:17]: Add the one in jobs too please 😄

File: linkedin.txt [hunk 3]
```diff
@@ -11,3 +11,17 @@ www.linkedin.com##.premium-upsell-link--extra-long
...
 www.linkedin.com##.msg-overlay-container-reflow.msg-overlay-container
 www.linkedin.com##.mb2.artdeco-card.community-panel
+
+! Hide LinkedIn Pay for premium under the profile in Feed page
+www.linkedin.com##.text-align-left.link-without-hover-state.t-bold.t-black.t-12.feed-identity-module__anchored-widget--premium-upsell.feed-identity-module__anchored-widget.link-without-visited-state.app-aware-link
+
+! Hide LinkedIn  "Try Premium for free" top bar
+www.linkedin.com##.premium-upsell-link--truncate.global-nav__spotlight-upsell.global-nav__primary-link--premium.global-nav__primary-link.link-without-visited-state
+
+! Hide LinkedIn "Try Premium for free" in My Network ! NOTE: loading this page is slower if enabled
```
Comment (mig4ng) [inline linkedin.txt]: Does the whole linkedin page get slower, or just the My Network?

**Review by mig4ng — CHANGES_REQUESTED** [review]

File: linkedin.txt (hunk 3 above)
Comment (mig4ng) [reply linkedin.txt]: I could not reproduce the slowdown

File: linkedin.txt (hunk 1 above)
Comment (ZivSimchoni) [reply linkedin.txt]: Hey, I double-checked that, they're visible to me, so I'll just add them at my filters (also w/ Firefox)

File: linkedin.txt (hunk 3 above)
Comment (ZivSimchoni) [reply linkedin.txt]: You're right, it has no slowdown, I guess it was my internet at the moment.

File: linkedin.txt (hunk 2 above)
Comment (ZivSimchoni) [reply linkedin.txt:17]: @mig4ng please check the rules again, I've added the jobs section and removed those you said are redundant for you.

File: linkedin.txt (hunk 1 above)
Comment (mig4ng) [reply linkedin.txt]: Feel free to keep them, or open a new PR with them and I will test again later.

**Review by mig4ng — APPROVED** [review]
Thanks for the contribution @ZivSimchoni ❤️ 

Feel free to keep add the premium filters if they are still bothering you via another pr :)

**Comment by ZivSimchoni** [comment]
Sure thing 👍🏽

**Pull request merged**